### ISSUE TITLE
UI Slider not working properly

### ISSUE THREAD

**Issue by RixBai** [surveyjs/widgets#181]
### What is the current behavior?
UI slider changes in parallel
![Recording surveyjs/survey-library#14](https://user-images.githubusercontent.com/43899212/81232708-5a9f3780-8fc3-11ea-8910-bec8ed2ecd59.gif)

### What is the expected behavior?
UI slider should be working separately.

### Specify your

* browser: Chrome
* browser version: Version 81.0.4044.138
* surveyjs platform (angular or react or jquery or knockout or vue): React
* surveyjs version: 1.7.5

**Comment by tsv2013** [comment]
Transferred to the corresponding repo

**Comment by tsv2013** [comment]
Looks like a duplicate of the https://github.com/surveyjs/widgets/issues/167

Answered here - https://surveyjs.answerdesk.io/ticket/details/t3714/multiple-nouislider-widgets-overwrite-each-others-selected-value

Answer in short - most likely you are using the same question name or the same valuse name for both questions.

If not - share your JSON

**Comment by RixBai** [comment]
Hi @tsv2013 
We already resolved the issue.
There was a problem on the json.
Thank you!
